pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -124,34 +164,57 @@ class TelegramHandler(EventHandler):
     def __init__(self, bot, config):
         self.bot = bot
         self.tbot = None
-        self.master = config.get('master', None)
+        master = config.get('master', None)
         self.pokemons = config.get('alert_catch', {})
         self.whoami = "TelegramHandler"
         self.config = config
+        if master == None:
+            self.master = None
+            return
+
+        with self.bot.database as conn:
+            # if master is not numeric, try to fetch it from the database
+            if not re.match(r'^[0-9]+$', str(master)): # master is numeric
```

Comment:
> 0.973 usec per loop

This is 0.973 **micro**second, if the abbreviations did not change in the last years. You would need to run it 1 million times in order to waste a whole second (micro = 1/1000000).